src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
```

Comment:
Doesn't require a change, but FYI you can import named components from the category folder index if you prefer:
```jsx
import { Portal } from '../../addons'
```